implementations/rust-boon/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "bowtie-rust-boon"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
Can you add this directory to .github/dependabot so it's kept autoupdated? It should go in 2 places, both to keep the `Dockerfile` updated and also your lib.

And also, can you add the directory to `.pre-commit-config.yaml` which should just ensure that `cargo fmt` is run on the rust file? You'll see `rust-jsonschema` already in there.

Reply:
done